staging/gatekeeper/patch/patch_2_upgrade_crds_job.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This patch adds mesosphere specific crd upgrade .
+
+source $(dirname "$0")/helpers.sh
+
+set -x
+
+SRCFILE="${BASEDIR}"/templates/crds.yaml
+
+sed -i '' -e '/# Create mesosphere value entries/,$d' ${SRCFILE}
```

Comment:
this won't work on linux btw :p maybe changing the `README` to tell devs to run `patch.sh` in the container is best? that way we don't need to think about OS diffs.

Reply:
yeah thats definitely an improvement! something for the near future, but i thought this does work on linux?

Reply:
this was taken from the promtheus-operator chart, fyi

Reply:
this `SEDI` pattern is all over our makefiles :p.... ex. https://github.com/mesosphere/konvoy/blob/e1bd7668650c9adc98405d94dbeb8f1971476b12/auto-provisioning/make/platform.mk#L17

Reply:
but sure, we can change it later. let the linux guys see it later :p